src/ontology/uberon-edit.obo

```diff
@@ -224033,6 +224033,17 @@ is_a: UBERON:0000029 ! lymph node
 relationship: dc-contributor http://orcid.org/0000-0002-1773-2692
 property_value: dcterms-date 2022-05-26T11:59:51Z xsd:dateTime
 
+[Term]
+id: UBERON:8470002
+name: septomarginal trabecula
+def: "A muscular trabecula in the right ventricle of the heart that links the lower part of the interventricular septum to the wall and to the base of the anterior papillary muscle of the right ventricle. The morphology and structure of the septomarginal trabecula varies in different species." [https://doi.org/10.1016/j.morpho.2010.03.004, https://doi.org/10.19080/APBIJ.2017.02.555595, https://orcid.org/0000-0001-7655-4833, PMID:30984450]
```

Comment:
>The morphology and structure of the septomarginal trabecula varies in different species.

Not 100% sure this is useful in the def and might belong more in the comment, but I think its no the worst thing to have here

Reply:
My preference would be to change the DOIs to PMID, but small thing. If using doi, probably better to use DOI: instead of the whole link. Next version of protege will resolve it, and DOI is registered in bioregistry and all.

Reply:
> My preference would be to change the DOIs to PMID, but small thing. If using doi, probably better to use DOI: instead of the whole link. Next version of protege will resolve it, and DOI is registered in bioregistry and all.

I don't see PMIDs for these references. I prefer the whole link for DOIs as it allows 1-click to access the reference.

Reply:
https://pubmed.ncbi.nlm.nih.gov/20359929/ for https://doi.org/10.1016/j.morpho.2010.03.004
https://pubmed.ncbi.nlm.nih.gov/22348232/ for https://doi.org/10.19080/APBIJ.2017.02.555595
DOI resolving will happen in next version of protege, and we try to use namespaces stuff where possible to keep things neat :) but yeah we will probably do a full programmatic change for DOI with the next update (I hope)

Reply:
> https://pubmed.ncbi.nlm.nih.gov/20359929/ for https://doi.org/10.1016/j.morpho.2010.03.004
> https://pubmed.ncbi.nlm.nih.gov/22348232/ for https://doi.org/10.19080/APBIJ.2017.02.555595

Agree and will replace the first line.
The second line are not the same.

Reply:
whoops, my fault for multitasking lol

Reply:
btws could we avoid using the 2nd citation please? Publisher is potentially predatory and I'd rather not give them visibility :)

Reply:
> btws could we avoid using the 2nd citation please? Publisher is potentially predatory and I'd rather not give them visibility :)

I don't know anything about that and used the reference when creating the class as it was provided by the requester. If you want to remove it, please do so in a separate ticket and communicate your intention with the requester (@emquardokus).

Reply:
I don't think a separate ticket is correct as it is pertinent to the new term addition. I see that @emquardokus is already a reviewer, so maybe best to have a discussion on if we can use another citation here instead.

> I don't know anything about that

see https://beallslist.net/ 

![Screenshot 2022-12-13 at 16 48 02](https://user-images.githubusercontent.com/76212760/207393806-c4c65e53-314b-428b-98b0-c4b33e44d63a.png)

Reply:
This website is run by an anonymous source and provides no evidence for its accusations. Any publisher "may be" (insert adjective here) or is "potentially" (insert adjective here).

@emquardokus, appreciate your feedback here.

Reply:
Bealls list is pretty accepted by academics, though they do have frontiers in, which is very on the fence lol. I mean its a personal preference to avoid, but if its too much trouble just keep it in. Journals being predatory does not = paper not being reliable or good.

Reply:
Thanks for sharing this, @shawntanzk. It's not an issue of being too much trouble; rather, there does not seem to be compelling evidence which detracts from the validity of this particular reference.

Reply:
Don't have an issue with the reference itself, just the journal. It is hard to draw a line on if a journal is predatory so its tricky. I mean even nature does participate in some practices I would consider predatory (e.g. scientific reports) , yet these journals have really good papers in them. I think its partly my "spider senses" that tingled so I went to look the publisher up, and bealls list tends to be the first place to do so. Anyway here's the wiki page that might have some more evidence (with citations in them) https://en.wikipedia.org/wiki/Juniper_Publishers.

Reply:
Thanks @shawntanzk, I'd never seen the Bealls list before, good to know!  I was primarily trying to find comparative anatomy since uberon is a multi-species ontology and this usually comes up in discussion if there is any questions.  If you think the journal publisher is too sketchy, please remove; however, it was the most comprehensive comparative anatomy paper I could find.  @bvarner-ebi Here's a different one to pigs, if you wanted to substitute: Gulyaeva AS, Roshchevskaya IM. Morphology of moderator bands (septomarginal trabecula) in porcine heart ventricles. Anat Histol Embryol. 2012 Oct;41(5):326-32. doi: 10.1111/j.1439-0264.2012.01139.x. Epub 2012 Feb 20. PMID: 22348232. 
I'm Okay with either choice here.